### DISCUSSION TITLE
Any way to write triplets?

### DISCUSSION THREAD

**Discussion by Djard** [hydrogen-music/hydrogen#1737] [General]
I need to write in triplets. Hydrogen does not support 12/8 time (only goes to 7 beats), and writing 6/8 half-bars does not permit view of a completed measure. Is there a workaround to write four triplets per bar?

**Comment by theGreatWhiteShark** [comment]
Have you tried setting the resolution to triplets? These options are located in the lower half of the resolution combo box on top of the pattern editor

**Reply by Djard** [reply]
Thanks. I updated the synth to version 1.2 beta and now am able to write triplets, and in 12/8, which for me was not possible in v1.1.1. Again, thanks for lending a kind hand.
